contracts/external/oracles/DaiPriceOracle.sol

```diff
@@ -221,7 +224,7 @@ contract DaiPriceOracle is
 
         uint256 targetPrice = getMidValue(
             EXPECTED_PRICE,
-            getOasisPrice(ethUsd).value,
+            getCurvePrice().value,
             getUniswapPrice(ethUsd).value
```

Comment:
nit: should we maybe just return a `uint256` instead of `Monetary.Price` from these functions? seems unnecessary to wrap and unwrap the values like this